src/routes/bkpr/+page.svelte

```diff
@@ -2,7 +2,7 @@
   import BackButton from '$lib/elements/BackButton.svelte'
   import RoutingPerformance from './components/RoutingPerformance.svelte'
   import RoutingFeesHistory from './components/RoutingFeesHistory.svelte'
-  import ChannelInsights from './components/ChannelInsights.svelte'
+  // import ChannelInsights from './components/ChannelInsights.svelte'
```

Comment:
Removing this component for now as it requires a refactor to match the new `Channel` and `Forward` types which we will likely do in v2 anyway.